maths/rkf45.py

```diff
@@ -9,43 +9,43 @@ class RangeError(Exception):
 
 def runge_futta_fehlberg_45(
     ode: Callable,
-    y0: float,
-    x0: float,
+    x_initial: float,
+    y_initial: float,
```

Comment:
These two were swapped (x before y) so please make sure that the values are also swapped in your doctests